src/wallet/fees.cpp

```diff
@@ -59,6 +59,8 @@ CAmount GetMinimumFee(unsigned int nTxBytes, const CCoinControl& coin_control, c
         if (fee_needed < min_mempool_fee) {
             fee_needed = min_mempool_fee;
             if (feeCalc) feeCalc->reason = FeeReason::MEMPOOL_MIN;
+        } else {
+            LogPrintf("Minimum mempool fee(%d) not met", min_mempool_fee);
```

Comment:
I don't think `Get*` is a good place to log a message.

What I meant is that in the current place where you see "mempool min fee not met" being logged, it should log the debug message of the state too.

Reply:
Thanks for the review, I'll do the changes.